packages/consent/consent-tools/src/types/wrapper.ts

```diff
@@ -13,6 +13,7 @@ export interface AnalyticsBrowserSettings {
 export interface InitOptions {
   updateCDNSettings?(cdnSettings: CDNSettings): CDNSettings
   disable?: boolean | ((cdnSettings: CDNSettings) => boolean)
+  initialPageview?: boolean
 }
```

Comment:
why is this object completely detached from the interface of the options of the analytics client? shouldn't they be the same?